pkgs/tools/security/echidna/default.nix

```diff
@@ -1,36 +1,31 @@
 { lib
```

Comment:
Isn't this mostly auto generated?

Reply:
good question, it doesn't seem to me so, I will check with the maintainer.